src/plone/volto/default_homepage/demo.py

```diff
@@ -349,271 +349,560 @@
         "title": "Welcome to Plone 6!",
         "description": "Congratulations! You have successfully installed Plone.",
```

Comment:
@stevepiercy I would change the description too; it doesn't make sense for a demo site